toolkit/tools/imagegen/configuration/user.go

```diff
@@ -87,6 +91,16 @@ func (p *User) UIDIsValid() (err error) {
 	return
 }
 
+// This function does not account for the Password key being intentionally
+// unspecified versus set to blank
+// PasswordIsValid returns an error if the User password is empty
```

Comment:
Add a note that root password being blank is acceptable and why we are accepting it